samples/jobs.py

```diff
@@ -230,11 +229,11 @@ def on_update_job_execution_rejected(rejected):
     host_resolver = io.DefaultHostResolver(event_loop_group)
     client_bootstrap = io.ClientBootstrap(event_loop_group, host_resolver)
 
-    if args.use_websocket == True:
-        proxy_options = None
-        if (args.proxy_host):
-            proxy_options = http.HttpProxyOptions(host_name=args.proxy_host, port=args.proxy_port)
+    proxy_options = None
+    if (args.proxy_host):
+        proxy_options = http.HttpProxyOptions(host_name=args.proxy_host, port=args.proxy_port)
 
+    if args.use_websocket == True:
         credentials_provider = auth.AwsCredentialsProvider.new_default_chain(client_bootstrap)
         mqtt_connection = mqtt_connection_builder.websockets_with_default_aws_signing(
             endpoint=args.endpoint,
```

Comment:
in this branch, change ~~websocket_proxy_options=proxy_options,~~ -> http_proxy_options=proxy_options